src/components/crossForm/CrossForm.vue

```diff
@@ -453,7 +453,8 @@ export default {
       return this.selectedToken?.receiveToken
     },
     receiveAmount() {
-      return this.amount - this.amount * this.fee
+      let bgAmount = new BigNumber(this.amount);
```

Comment:
```suggestion
      const bgAmount = new BigNumber(this.amount);
```
Try to use const when posible